cmd/kubeadm/app/cmd/init.go

```diff
@@ -323,6 +323,17 @@ func (i *Init) Run(out io.Writer) error {
 
 	adminKubeConfigPath := filepath.Join(kubeConfigDir, kubeadmconstants.AdminKubeConfigFileName)
 
+	// ReadSentinelFile check sentinelfile exist
+	if data, err := kubeadmutil.ReadSentinelFile(); err != nil {
```

Comment:
@timothysc my idea was to handle both the case where `err` is an actual error and when data is valid.
i guess this can be improved in later comments.